src/ui/TimeTable.tsx

```diff
@@ -303,18 +296,6 @@ const TimeTable = ({ id, className, onChange, value, ...props }: Props) => {
     };
   };
 
-  useEffect(() => {
-    if (!value?.dateStart || !value?.dateEnd) {
-      const nextWeekWednesday = nextWednesday(new Date());
-      onChange({
-        data: value?.data ?? {},
-        dateStart: formatDate(nextWeekWednesday),
-        dateEnd: formatDate(nextWeekWednesday),
-      });
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
```

Comment:
Why was this removed, isn't this needed?

Reply:
I've moved this to the defaultValues of the form.